acinclude.m4

```diff
@@ -273,8 +273,7 @@ AC_DEFUN([RJK_SIZE_MAX],[
 
 AC_DEFUN([RJK_GETOPT],[
   AC_CHECK_FUNC([getopt_long],[],[
-    AC_LIBOBJ([getopt])
```

Comment:
This doesn't seem right. The getopt implementation is there for platforms that don't have it, and is only compiled if it's not present in the platform's C library.

Reply:
Without removing these files, I get the following error in *Termux*:

```
$ ./autogen.sh
$ ./configure
$ make check
...
./getopt.h:137:12: error: a function declaration without a prototype is deprecated in all versions of C and is treated as a zero-parameter prototype in C2x, conflicting with a previous declaration [-Werror,-Wdeprecated-non-prototype]
extern int getopt ();
           ^
...
```

Reply:
Alright I figured out how to build it on *Termux* while keeping these files. The issue is that *Termux*'s `/data/data/com.termux/files/usr/include/features.h` is from *AOSP* and is quite brief. It doesn't define `__GNU_LIBRARY__`. Defining it manually skips the `getopt*` files you include, and builds:

```
$ ./autogen.sh
$ ./configure CPPFLAGS=-D__GNU_LIBRARY__=6
$ make clean
$ make check
```

Reply:
Indeed, not every platform has `getopt_long`. You'll find the same approach in a lot of GNU software. 

However it's not good that we try to include the included `getopt.h` even when a suitable one is available in the C library. I'll see if I can do something about that.